components/org.wso2.carbon.identity.api.server.application.management/org.wso2.carbon.identity.api.server.application.management.v1/src/main/resources/applications.yaml

```diff
@@ -3315,6 +3319,30 @@ components:
           type: string
           example: basic
 
+    IdpAppRoleConfigurations:
```

Comment:
Shall we rename this to `AppRoleConfigurations`?
I feel that `IdpAppRoleConfigurations` is not self explaining

Reply:
Aligns with the naming in identity framework as well.
Addressed in 8cb8369a51d879d11364fe26a380425f7670bff7